bazel-bep/src/lib.rs

```diff
@@ -45,6 +45,8 @@ pub mod types {
     }
 }
 
+// NOTE: `build_event_stream(.serde).rs` has some failing clippy lints
+#[allow(clippy::all)]
```

Comment:
`trunk.yaml` doesn't seem to be able to ignore these generated files (perhaps because Clippy reports absolute paths?)

Reply:
I would expect us to handle this properly by adding a lint.ignore, but fine for now

Reply:
Exactly my thoughts. I added one and it didn't work
https://github.com/trunk-io/analytics-cli/pull/256/files#diff-45a5a35dd7ad9a52d0fd3540f6c0d16a763575de51cc9a0a6c6f6a9f1da62ecdR78-R83